gig-marketplace.md

```diff
@@ -0,0 +1,58 @@
+## Title
+* Create an intranet marketplace where InnerSource project owners and potential contributors can easily discover one another
+
+## Patlet
+Establish a marketplace by creating a website(s) that lists InnerSource project needs as "gigs". 
+You can attract potential contributors by posting gigs with explicit requirements (amount of time, skills). People can sign up for a gig and garner approval from their management for those explicit requirements. Individual contributors are then awarded points for the gigs they complete. 
+This makes it explicit what the commitment is and that management agrees with it while encouraging and tracking involvement.
+
+## Problem
+There is no single, uniform way for InnerSource project owners and potential contributors to discover each other and connect.
+
+## Story (Optional)
+Your InnerSource program has been established and there is great interest but little actual activity. 
+Individual project owners are encountering difficulties attracting a community of contributors.
+Most managers are supportive in allowing their employees the time to involve themselves in an InnerSource project. 
+There simply is no single uniform way for contributors to discover all the InnerSource opportunities that exist that they might be interested in contributing to.
+Conversely, innerSource project owners also have no way of informing a general audience about their projects and its goals.
+
+## Context
+* There are multiple business units in your organization that do not communicate well.
+* Individuals have no way of discovering what InnerSource opportunities exist across the entire company.
+* InnerSource project owners have no way of advertising their project to potential contributors.
+* Individuals have been given some time by their management to get involved in InnerSource projects of their choosing.
+* Managers have no way of knowing what the true time/effort commitment will be if their employee chooses to take on tasks to support an InnerSource project.
```

Comment:
from Georg: move this to the _forces_ section
from @rrrutledge : suggestion for new context: "Managers are responsible to track and reward people's time"
from all: let's put this in, too.